contracts/misc/Imports.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity 0.4.24;
```

Comment:
Maybe leaving it in `contracts/test/Imports.sol`, since I think we only need this during testing to set up the test environment.